### PR TITLE
fix readme.md

### PR DESCRIPTION
**Pull request by Naoki-Hiraoka**
* The description of `hrpsys_gazebo_tutorials` was moved from `rtmros_gazebo` https://github.com/start-jsk/rtmros_gazebo/pull/251
* Install method was updated https://github.com/start-jsk/rtmros_gazebo/issues/247#issuecomment-610945439 https://github.com/start-jsk/rtmros_tutorials/pull/567#pullrequestreview-390490568

### CODE REVIEW

**Comment by Naoki-Hiraoka** [comment]
@pazeshun Could you review this?

File: README.md [hunk 1]
```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
...
+
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
+    - *&lt;robot_name&gt;*.yaml for configurating gazebo setting and hrpsys setting
+    - (automatically generated) *&lt;robot_name&gt;*.urdf under robot_models/*&lt;robot_name&gt;* directory
+    - (automatically generated) hrpsys settings (you should have a collada or VRML robot model file)
+    - *&lt;robot_name&gt;*_optional_urdf_setting.sh under robot_models/*&lt;robot_name&gt;* directory, this is for adding description used by gazebo (such as sensor settings, collision and friction setting)
+
+#### (automatically generated files)
+
+You can use robot_models/install_robot_common.sh for installing urdf model file. This scripts converts collada file in [hrpsys_ros_bridge_tutorials]/models directory to urdf file.
```
Comment (pazeshun) [inline README.md:54]: I couldn't find `robot_models/install_robot_common.sh`.

File: README.md [hunk 2]
```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
...
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+### [hrpsys_gazebo_tutorials]
+
+This package is a collection of examples for using hrpsys_gazebo system and utility scripts.
+
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
```
Comment (pazeshun) [inline README.md:46]: Is this section additional? In my understanding, this package can be used with SampleRobot in default, and that procedure doesn't require explicit preparation of robot model by user.
Also, you should first say where to put model file and how to use it in gazebo simulation.
After that, you can explain about details (what is automatically generated, etc.)
In my understanding, when `rtmros_hrp2` is downloaded, `catkin build` is only needed to use HRP2, isn't it?
So I think you should explain as follows:
1. Explain SampleRobot (only `catkin build` is needed)
2. Explain HRP2 (downloading `rtmros_hrp2` and `catkin build` are needed)
3. Explain user-defined robot (placing model file to right place, running script, and `catkin build` are needed)
4. Explain about details (what is automatically generated, etc.)

File: README.md [hunk 3]
```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
 
 ### Package Description
 
-hrpsys_ros_bridge_tutorialsls
+### [hrpsys_ros_bridge_tutorials]
```
[the remark this answers is not in the document]
Comment (pazeshun) [reply README.md:36]: I'm not sure why this writing works on original README

File: README.md [hunk 4]
```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
...
 --------------
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+### [hrpsys_gazebo_tutorials]
```
[the remark this answers is not in the document]
Comment (pazeshun) [reply README.md:42]: I'm not sure why this writing works on original README

File: README.md (hunk 2 above)
Comment (pazeshun) [reply README.md:46]: I understand this is copied from `rtmros_gazebo`, but it's hard to understand...

File: README.md [hunk 5]
```diff
@@ -4,30 +4,12 @@ rtmros_tutorials  [![Build Status](https://travis-ci.org/start-jsk/rtmros_tutori
...
-catkin_make --only-pkg-with-deps hrpsys_gazebo_tutorials
-source ~/ros/ws_rtmros_tutorials/devel/setup.bash # this line is necessary for bug, need to be fixed.
-catkin_make --only-pkg-with-deps hrpsys_gazebo_tutorials --force-cmake # this line is necessary for bug, need to be fixed.
-catkin_make --only-pkg-with-deps hrpsys_gazebo_tutorials --force-cmake # this line is necessary for bug, need to be fixed.
-# add setting
-echo "source ~/ros/ws_rtmros_tutorials/devel/setup.bash" >> ~/.bashrc
-echo "source \`rospack find hrpsys_gazebo_tutorials\`/setup.sh" >> ~/.bashrc
-```
+
+Please refer [rtmros_common] for installing these packages.
```
[the remark this answers is not in the document]
Comment (Naoki-Hiraoka) [reply README.md:8]: Thank you. I fixed it.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Naoki-Hiraoka) [reply README.md:36]: Thank you. I fixed it.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Naoki-Hiraoka) [reply README.md]: Thank you. I fixed it.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Naoki-Hiraoka) [reply README.md:42]: Thank you. I fixed it.

File: README.md (hunk 1 above)
Comment (Naoki-Hiraoka) [reply README.md]: Me too. (This is simply copied from rtmros_gazebo)
I removed this section.

File: README.md (hunk 2 above)
Comment (pazeshun) [reply README.md:42]: I understand this section is for `hrpsys_ros_bridge_tutorials`. Now my comment seems meaningless

File: README.md [hunk 6]
```diff
@@ -51,9 +33,22 @@ roseus hrp2jsknt-interface.l
...
 
-hrpsys_ros_bridge_tutorialsls
+### [hrpsys_ros_bridge_tutorials]
 --------------
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
```
Comment (pazeshun) [inline README.md:42]: Maybe we cannot get what we should actually do from `You should prepare robot model file`.
How about noting actual operation (for example, `You should prepare robot model file (e.g., no additional operation is required for SampleRobot, compiling hrp2_models is required for HRP2)`)

Comment (Naoki-Hiraoka) [reply README.md]: Thank you. I fixed it.

**Pull request merged**